code/game/objects/structures/tables_racks.dm

```diff
@@ -38,7 +39,8 @@
 	. = ..()
 	if(!istype(H) || H.shoes || !(H.mobility_flags & MOBILITY_STAND))
 		return ..()
-	if(prob(5))
+	if((world.time >= last_bump + 100) && prob(5))
+		last_bump = world.time

```

Comment:
I'm debating putting this line outside of the check, but that could make toe stubbing *too* rare.